src/renderer/Slider.tsx

```diff
@@ -147,29 +150,41 @@ export default function Slider() {
         ref={increaseButtonRef}
         role="button"
         aria-label="Increase pre-amplification gain"
-        className="slider-top"
+        className="center slider-top"
         onMouseDown={() => handleArrowInput(true)}
         onMouseUp={stopIncrement}
         onKeyDown={(e) => listenForEnter(e, () => handleDeltaChangeGain(true))}
         tabIndex={0}
-      />
+      >
+        <SliderArrowIcon type="up" />
+      </div>
       <input
         type="range"
         min={MIN}
         max={MAX}
         value={preAmpGain}
         onChange={handleRangeInput}
+        style={
+          // Set css variables for determining upper/lower track
+          {
+            '--min': MIN,
```

Comment:
I guess theres no way to access the min, max and value attribute of the input element from the css side?